schema/manifest-list-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "description": "OpenContainer Image Manifest List Specification",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://opencontainers.org/schema/image/manifest-list",
+  "type": "object",
+  "properties": {
+    "schemaVersion": {
+      "description": "This field specifies the image manifest-list schema version as an integer",
+      "id": "https://opencontainers.org/schema/image/manifest-list/schemaVersion",
+      "type": "integer"
+    },
+    "mediaType": {
+      "id": "https://opencontainers.org/schema/image/manifest-list/mediaType",
+      "$ref": "defs-image.json#/definitions/mediaType"
+    },
+    "manifests": {
+      "type": "array",
+      "items": {
+        "$ref": "defs-image.json#/definitions/manifestObject"
+      }
+    },
+    "annotations": {
```

Comment:
oh oops. That is yank-and-paste cruft I was mulling on.

On Fri, Apr 15, 2016 at 2:50 PM Stephen Day <notifications@github.com>
wrote:

> In schema/manifest-list-schema.json
> <https://github.com/opencontainers/image-spec/pull/32#discussion_r59946235>
> :
>
> > +    "schemaVersion": {
> > +      "description": "This field specifies the image manifest-list schema version as an integer",
> > +      "id": "https://opencontainers.org/schema/image/manifest-list/schemaVersion",
> > +      "type": "integer"
> > +    },
> > +    "mediaType": {
> > +      "id": "https://opencontainers.org/schema/image/manifest-list/mediaType",
> > +      "$ref": "defs-image.json#/definitions/mediaType"
> > +    },
> > +    "manifests": {
> > +      "type": "array",
> > +      "items": {
> > +        "$ref": "defs-image.json#/definitions/manifestObject"
> > +      }
> > +    },
> > +    "annotations": {
>
> Is this part of the specification, yet?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/32/files/a5632104014382df2fc80ac79f88c26e0b286a4a#r59946235>
>